packages/dd-trace/src/appsec/iast/taint-tracking/secure-marks.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const vulnerabilities = require('../vulnerabilities')
+const { getNextSecureMark } = require('./secure-marks-generator')
+
+const marks = {}
+Object.keys(vulnerabilities).forEach(vulnerability => {
+  marks[vulnerability + '_MARK'] = getNextSecureMark()
+})
+
+let [asterisk, ...rest] = Object.values(marks)
+rest.forEach(mark => { asterisk |= mark })
```

Comment:
I should have to read this code multiple times to understand it...

Here suggestions that I think are more readable.

```suggestion
let asterisk = 0x0
Object.values(marks).forEach(mark => { asterisk |= mark })
```
Or
```suggestion
const asterisk = Object.values(marks).reduce((total, current) => total | current, 0x0)
```